specification/advisor/resource-manager/Microsoft.Advisor/stable/2022-10-01/advisor.json

```diff
@@ -0,0 +1,1762 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-10-01",
+    "title": "AdvisorManagementClient",
+    "description": "REST APIs for Azure Advisor"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Advisor/metadata/{name}": {
+      "get": {
+        "tags": [
+          "Metadata"
+        ],
+        "summary": "Gets the metadata entity.",
+        "operationId": "RecommendationMetadata_Get",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of metadata entity.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully retrieved metadata entities",
+            "schema": {
+              "$ref": "#/definitions/MetadataEntity"
+            }
+          },
+          "404": {
+            "description": "Client sent unknown metadata name",
+            "schema": {
+              "$ref": "#/definitions/ARMErrorResponseBody"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ArmErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetMetadata": {
+            "$ref": "./examples/GetRecommendationMetadataEntity.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Advisor/metadata": {
```

Comment:
We prefer to pluralize the segment for the list. Can you make it entities instead of metadata. This might need a change on the GET call above as well

Reply:
@sjanamma : The current 2022-10-01 manifest for arm has been deployed with the above path. Advisor will make sure to pluralize the above spec in the future arm / swagger updates. Tagging @jportugal0 for visibility

Reply:
Thanks for the review - Advisor team has this comment taken as PBI to include the change suggested here : https://msazure.visualstudio.com/One/_backlogs/backlog/AzureAdvisor/Features/?workitem=17048171 and will introduce the change needed by the comment in the next api-version increment of advisor.